### PR TITLE
Bug/1534 execution history fails sample jobs

### LINKED ISSUE #1534: Monitor: execution history of "Sample custom job" fails
When I go to the Scheduling page and click "More"/"Execution history" for "Sample custom job", I get following error:
```
16:17:10.011 [http-nio-8080-exec-5] WARN  o.d.monitor.server.SecureGwtServlet - Unexpected exception occurred in GWT servlet: com.google.gwt.user.server.rpc.UnexpectedException
com.google.gwt.user.server.rpc.UnexpectedException: Service method 'public abstract java.util.List org.datacleaner.monitor.scheduling.SchedulingService.getAllExecutions(org.datacleaner.monitor.shared.model.TenantIdentifier,org.datacleaner.monitor.shared.model.JobIdentifier) throws org.datacleaner.monitor.shared.model.DCSecurityException' threw an unexpected exception: java.lang.IllegalStateException: Failed to parse ExecutionIdentifier document: /demo/results/Sample custom job-1455267630289.analysis.execution.log.xml
	at com.google.gwt.user.server.rpc.RPC.encodeResponseForFailure(RPC.java:415) ~[gwt-servlet-2.7.0.jar:na]
	at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:605) ~[gwt-servlet-2.7.0.jar:na]
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:333) ~[gwt-servlet-2.7.0.jar:na]
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:303) ~[gwt-servlet-2.7.0.jar:na]
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373) ~[gwt-servlet-2.7.0.jar:na]
	at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62) ~[gwt-servlet-2.7.0.jar:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648) [servlet-api.jar:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729) [servlet-api.jar:na]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291) [catalina.jar:8.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206) [catalina.jar:8.0.30]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239) [catalina.jar:8.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206) [catalina.jar:8.0.30]
	at org.ocpsoft.rewrite.servlet.RewriteFilter.doFilter(RewriteFilter.java:205) [rewrite-servlet-2.0.12.Final.jar:2.0.12.Final]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239) [catalina.jar:8.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206) [catalina.jar:8.0.30]
	at org.datacleaner.monitor.server.GwtCacheControlFilter.doFilter(GwtCacheControlFilter.java:70) [DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239) [catalina.jar:8.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206) [catalina.jar:8.0.30]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:330) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:118) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:84) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:113) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:103) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:113) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter.doFilter(RememberMeAuthenticationFilter.java:146) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:154) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:45) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.authentication.www.BasicAuthenticationFilter.doFilter(BasicAuthenticationFilter.java:150) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:199) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:110) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:50) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107) [spring-web-4.1.9.RELEASE.jar:4.1.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:87) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:192) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160) [spring-security-web-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:344) [spring-web-4.1.9.RELEASE.jar:4.1.9.RELEASE]
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:261) [spring-web-4.1.9.RELEASE.jar:4.1.9.RELEASE]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239) [catalina.jar:8.0.30]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206) [catalina.jar:8.0.30]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212) [catalina.jar:8.0.30]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106) [catalina.jar:8.0.30]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502) [catalina.jar:8.0.30]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141) [catalina.jar:8.0.30]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79) [catalina.jar:8.0.30]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:616) [catalina.jar:8.0.30]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88) [catalina.jar:8.0.30]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:521) [catalina.jar:8.0.30]
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1096) [tomcat-coyote.jar:8.0.30]
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:674) [tomcat-coyote.jar:8.0.30]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1500) [tomcat-coyote.jar:8.0.30]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1456) [tomcat-coyote.jar:8.0.30]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_72]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_72]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.0.30]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_72]
Caused by: java.lang.IllegalStateException: Failed to parse ExecutionIdentifier document: /demo/results/Sample custom job-1455267630289.analysis.execution.log.xml
	at org.datacleaner.monitor.server.jaxb.SaxExecutionIdentifierReader.parse(SaxExecutionIdentifierReader.java:100) ~[DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	at org.datacleaner.monitor.server.jaxb.SaxExecutionIdentifierReader.read(SaxExecutionIdentifierReader.java:66) ~[DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	at org.datacleaner.monitor.server.SchedulingServiceImpl$3$1.eval(SchedulingServiceImpl.java:623) ~[DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	at org.datacleaner.monitor.server.SchedulingServiceImpl$3$1.eval(SchedulingServiceImpl.java:620) ~[DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	at org.datacleaner.repository.file.FileRepositoryFile.readFile(FileRepositoryFile.java:149) ~[DataCleaner-engine-core-5.1.4-SNAPSHOT.jar:na]
	at org.datacleaner.monitor.server.SchedulingServiceImpl$3.eval(SchedulingServiceImpl.java:620) ~[DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	at org.datacleaner.monitor.server.SchedulingServiceImpl$3.eval(SchedulingServiceImpl.java:617) ~[DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	at org.apache.metamodel.util.CollectionUtils.map(CollectionUtils.java:284) ~[MetaModel-core-4.5.4.jar:4.5.4]
	at org.datacleaner.monitor.server.SchedulingServiceImpl.getAllExecutions(SchedulingServiceImpl.java:616) ~[DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_72]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_72]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_72]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_72]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:317) ~[spring-aop-4.1.9.RELEASE.jar:4.1.9.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190) ~[spring-aop-4.1.9.RELEASE.jar:4.1.9.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.1.9.RELEASE.jar:4.1.9.RELEASE]
	at org.springframework.security.access.intercept.aopalliance.MethodSecurityInterceptor.invoke(MethodSecurityInterceptor.java:64) ~[spring-security-core-3.2.9.RELEASE.jar:3.2.9.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.9.RELEASE.jar:4.1.9.RELEASE]
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:207) ~[spring-aop-4.1.9.RELEASE.jar:4.1.9.RELEASE]
	at com.sun.proxy.$Proxy74.getAllExecutions(Unknown Source) ~[na:na]
	at org.datacleaner.monitor.server.SchedulingServiceServlet.getAllExecutions(SchedulingServiceServlet.java:82) ~[DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_72]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_72]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_72]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_72]
	at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:587) ~[gwt-servlet-2.7.0.jar:na]
	... 66 common frames omitted
Caused by: org.xml.sax.SAXParseException: Premature end of file.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:177) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1437) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$PrologDriver.next(XMLDocumentScannerImpl.java:1019) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:606) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:510) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:848) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:777) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:643) ~[na:1.8.0_72]
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:327) ~[na:1.8.0_72]
	at javax.xml.parsers.SAXParser.parse(SAXParser.java:195) ~[na:1.8.0_72]
	at org.datacleaner.monitor.server.jaxb.SaxExecutionIdentifierReader.parse(SaxExecutionIdentifierReader.java:95) ~[DataCleaner-monitor-services-5.1.4-SNAPSHOT.jar:na]
	... 91 common frames omitted
```

### PR DESCRIPTION
**Pull request by ClaudiaPHI**
The changes handle the situation when 

- Execution log is not readable:  the log is empty because the job was interrupted
- There is an execution log, but no execution result file because the job ended prematurely

Fixes #1534

### CODE REVIEW

File: monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java [hunk 1]
```diff
@@ -647,7 +647,12 @@ public ExecutionIdentifier eval(final RepositoryFile file) {
                         ExecutionIdentifier result = file.readFile(new Func<InputStream, ExecutionIdentifier>() {
                             @Override
                             public ExecutionIdentifier eval(InputStream in) {
-                                return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
+                                try {
+                                    return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
+                                } catch (Exception e) {
+                                    logger.error("The file {} could not be read" + e, file.getQualifiedPath());
```
Comment (LosD) [inline monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java]: The errors level seems excessive. error is usually reserved for fatal problem, this is more of a problem that doesn't really affect the program.

**Comment by LosD** [comment]
At first glance, this seems pretty dangerous, as we're swallowing exceptions, but this is exactly what we want to do in this case.

... I'm a little afraid this might be too wide, though, as _any_ problem will trigger this. Can't we check specifically for an empty file instead?

File: monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java (hunk 1 above)
Comment (LosD) [inline monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java]: BTw, you should avoid concatenating like this in SLF4J, it should work fine to have the exception as the last argument, as in `logger.warn("The file {} could not be read", file.getQualifiedPath(), e);`

http://slf4j.org/faq.html#paramException

**Comment by ClaudiaPHI** [comment]
I cannot check if the file is empty before reading. The class "SaxExecutionIdentifierReader" we are using to parse the file throws exception because it could not find any properties.

**Comment by LosD** [comment]
Hmmm... You have the file handle, what's wrong with `boolean empty = file.length() == 0`?

**Comment by LosD** [comment]
Oh crap. It's a `RepositoryFile`. Checking if there's a way. Give me a moment.

**Comment by LosD** [comment]
Okay, there's `RepositoryFile.getSize()` but you'll need to just go ahead if it returns -1 (which can be the case if it's a DB-based repository).

**Comment by ClaudiaPHI** [comment]
I have added more text to the user so that it's visible there has been an error while running the job. Checking if the file is empty will not cover any other parsing exceptions while trying to read the file.

**Comment by LosD** [comment]
Agreed. @jhorcicka, are you happy? Then I'm happy :)

**Comment by jhorcicka** [comment]
When the result file is empty, it works fine, now:
empty-files

However, when the file is just not readable, I still get the error: 
not-readable-execution-history

File: monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java [hunk 2]
```diff
@@ -644,13 +644,20 @@ public ExecutionLog getLatestExecution(TenantIdentifier tenant, JobIdentifier jo
...
                     public ExecutionIdentifier eval(final RepositoryFile file) {
-                        ExecutionIdentifier result = file.readFile(new Func<InputStream, ExecutionIdentifier>() {
-                            @Override
-                            public ExecutionIdentifier eval(InputStream in) {
-                                return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
-                            }
-                        });
-                        return result;
+                        try {
+                            ExecutionIdentifier result = file.readFile(new Func<InputStream, ExecutionIdentifier>() {
```
Comment (jhorcicka) [inline monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java]: final?

File: monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java [hunk 3]
```diff
@@ -644,13 +644,20 @@ public ExecutionLog getLatestExecution(TenantIdentifier tenant, JobIdentifier jo
...
+                                @Override
+                                public ExecutionIdentifier eval(InputStream in) {
+                                    return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
+                                }
+                            });
+                            return result;
+                        } catch (Exception e) {
+                            logger.warn("The file " + file.getQualifiedPath() + " could not be read or parsed correctly"
+                                    + e);
```
Comment (jhorcicka) [inline monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java]: Separator (space, dot, colon, ...) at the end of the String: "...correctly_". I assume that "e" does not start with a space.

**Comment by jhorcicka** [comment]
Tested and LGTM. Let's merge it when Travis is happy.

File: monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java (hunk 3 above)
Comment (LosD) [inline monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java]: Hmmm, this one was missed. We should probably start using the review feature (I've also been resisting though, but that would avoid missing stuff)

Comment (LosD) [inline monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java]: No it wasn't. Just GitHub that changed how it presents old comments...

**Pull request merged**